scripts/bump.js

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+
+const { bold, underline } = require('chalk');
+const { spawnSync } = require('child_process');
+const inquirer = require('inquirer');
+const semver = require('semver');
+
+const REPO_DIR = path.dirname(__dirname);
+
+const readJson = path => {
+  return JSON.parse(fs.readFileSync(path, { encoding: 'utf8' }));
+};
+
+const writeJson = (path, obj) => {
+  const content = JSON.stringify(obj, null, 2) + '\n';
+  fs.writeFileSync(path, content, { encoding: 'utf8' });
+};
+
+const PACKAGE_ORIG_VERSIONS = [
+  'cli',
+  'core',
+  'schema',
+  'legacy-scripting-runner'
+].reduce((result, packageName) => {
+  const packageJsonPath = path.join(
+    REPO_DIR,
+    'packages',
+    packageName,
+    'package.json'
+  );
+  const packageJson = readJson(packageJsonPath);
+  result[packageName] = packageJson.version;
```

Comment:
It's totally fine here (and is not worth changing) but it's worth noting (in case you weren't aware) that writing a reduce this way modifies the accumulator object. Since we're passing in `{}` it doesn't matter. If we were instead acting on an existing variable, like this: 

```js
const originalSettings = {a: 1, b: 2}
const newSettings = ['c', 'd', 'e'].reduce((result, someVar) => {
  result[someVar] = true; 
  return result
}, originalSettings)

console.log(newSettings, originalSettings)
// => {a: 1, b: 2, c: true, d: true, e: true}, {a: 1, b: 2, c: true, d: true, e: true}
```

Instead, you'd use the spread operator to return a new object each time:

```js
const originalSettings = {a: 1, b: 2}
const newSettings = ['c', 'd', 'e'].reduce((result, someVar) => {
  return {...result, [someVar]: true}
}, originalSettings)

console.log(newSettings, originalSettings)
// => {a: 1, b: 2, c: true, d: true, e: true}, {a: 1, b: 2}
```

like I said, totally reasonable here. But, the more you know!

Reply:
I was aware of this but the spread operator is a good callout! Fixed in aed592d.